### PR TITLE
IDEA-950 - Add repo visibility to `github_repo`

### PR DESCRIPTION
**Pull request by austinkelleher**
| Property     | Type     |
| ------------ | -------- |
| `visibility` | `string` |

### CODE REVIEW

File: src/client/GraphQLClient/fragments.ts [hunk 1]
```diff
@@ -83,6 +83,7 @@ export default {
     pushedAt
     rebaseMergeAllowed
     url
+    visibility
```
Comment (austinkelleher) [inline src/client/GraphQLClient/fragments.ts:86]: Here was where the new property was added in the GraphQL request

File: src/sync/converters.ts [hunk 2]
```diff
@@ -403,6 +403,7 @@ export function toRepositoryEntity(data: OrgRepoQueryResponse): RepoEntity {
     lockReason: data.lockReason || '',
     mergeCommitAllowed: data.mergeCommitAllowed,
     rebaseMergeAllowed: data.rebaseMergeAllowed,
+    visibility: data.visibility,
```
Comment (austinkelleher) [inline src/sync/converters.ts:406]: Converter update

**Review by ndowmon — DISMISSED** [review]
❤️

File: jupiterone/questions/questions.yaml [hunk 3]
```diff
@@ -1,162 +1,175 @@
 ---
 sourceId: managed:github
-integrationDefinitionId: "${integration_definition_id}"
+integrationDefinitionId: '${integration_definition_id}'
```
Comment (austinkelleher) [inline jupiterone/questions/questions.yaml:3]: All of this was prettier formatting...

**Pull request merged**